.babelrc

```diff
@@ -1,6 +1,4 @@
 {
-    "presets": [
-        "@babel/preset-env",
-        "@babel/preset-react"
-    ]
-}
\ No newline at end of file
+  "presets": ["@babel/preset-env", "@babel/preset-react"],
```

Comment:
Btw, do you need preset env at all?

Reply:
"browserslist": [
    "last 1 chrome version"
  ],